velox/exec/tests/SharedArbitratorTest.cpp

```diff
@@ -3681,78 +3840,42 @@ TEST_F(SharedArbitrationTest, concurrentArbitration) {
         return false;
       });
```

Comment:
We don't need the fakeOperatorFactory_ setup in this tests? And we can make total capacity a smaller value. Then we might expect local oom from the queries? Either aborted or run query out of memory?